src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/sam/SwaggerSamClient.scala

```diff
@@ -30,6 +30,9 @@ class SwaggerSamClient(samBasePath: String, cacheExpiryTime: FiniteDuration, cac
   private val notebookClusterResourceTypeName = "notebook-cluster"
   private val billingProjectResourceTypeName = "billing-project"
 
+  private val projectOwnerPolicyName = "owner"
+  private val clusterCreatorPolicyName = "creator"
```

Comment:
I'm a little bothered by this: before we were querying Sam with an _action_ name using this API: https://sam.dsde-prod.broadinstitute.org/#!/Resources/resourceAction

Whereas now we're retrieving Sam _policy_ names via this API: https://sam.dsde-prod.broadinstitute.org/#!/Resources/listResourcesAndPolicies. While it works, it means that Leo needs to have knowledge about which actions map to which policy, which it didn't need to have before.

I'm wondering if it would be useful for Sam to expose an API like:
```
GET /api/resource/{resourceTypeName}/action/{action}
```
which gets all resources with the given type/action. I'm going to ask Project Team about this. :)

Reply:
Update: Doug said it was ok